src/aave_mapping.ts

```diff
@@ -27,6 +28,7 @@ export function handleProposalExecuted(event: StatusChangeToExecuted): void {
   if (proposal != null) {
     proposal.status = "Executed";
     proposal.timestamp = event.block.timestamp;
+    proposal.endDate = event.block.timestamp;
```

Comment:
end date should be set on queued or canceled, not executed event. Please update in all files